vizro-core/src/vizro/figures/_kpi_cards.py

```diff
@@ -137,7 +142,12 @@ def kpi_card_reference(  # noqa: PLR0913
     delta = value - reference
     delta_relative = delta / reference if reference else np.nan
 
-    header = dbc.CardHeader([html.P(icon, className="material-symbols-outlined") if icon else None, html.H2(title)])
+    header = dbc.CardHeader(
+        [
+            html.P(icon, className="material-symbols-outlined") if icon else None,
+            html.H4(title, className="card-kpi-title"),
```

Comment:
Change to `H4` is required as otherwise the KPI Cards look bad on all other bootstrap themes. Our font-sizes defined by the QB design library do not match up with Bootstrap's heading font-sizes. This should probably change in the future.